tests/src/image-smoke.spec.ts

```diff
@@ -120,4 +114,33 @@ describe('Image workflow verification', async () => {
     imagesPage = await imageDetailsPage.deleteImage();
     expect(await imagesPage.waitForImageDelete('build-test-image')).toBeTruthy();
   });
+
+  test('Prune images', async () => {
+    const imageList = ['quay.io/podman/image1', 'quay.io/podman/image2'];
+
+    let imagesPage = await navBar.openImages();
+    await playExpect(imagesPage.heading).toBeVisible();
+
+    for (const image of imageList) {
+      const pullImagePage = await imagesPage.openPullImage();
```

Comment:
I'm not sure what you mean by extraction?

Reply:
This whole code could be extracted into a function with parameters. You could also have a part where you pull an image and rename it, or it could be two individual functions.

Reply:
done